profile.cc

```diff
@@ -1,5 +1,12 @@
 #include "profile.h"
+#include <cmath>
 
-const double Profile::GetTime() {
-  return 0.0;
+const double Profile::GetTime(bool ignore_max_velocity) {
+  if (ignore_max_velocity) {
+    return std::sqrt((2*goal_.position) / kMaxVelocity);
+  } else {
+    int time_accelerating = (kMaxVelocity + current_.velocity) / kMaxAcceleration;
+    int time_constant = ((goal_.position - (time_accelerating * kMaxVelocity)) / kMaxVelocity);
```

Comment:
Should check if constant velocity is needed. You may end up following a triangle not a trapezoid. Also `double` not `int`